app/grandchallenge/evaluation/models.py

```diff
@@ -624,7 +624,10 @@ def clean(self):
         super().clean()
         self._clean_algorithm_submission_settings()
         self._clean_submission_limits()
-        self._clean_parent_phase()
+        try:
+            self._clean_parent_phase()
+        except ValidationError as e:
+            raise ValidationError({"parent": e})
```

Comment:
I think this is too fragile. It relies on the fact that the option is on the first tab and that the tab is displayed by default. For these settings all validation errors go to the top otherwise they're hidden. This is not just a problem for parent selection.

What are the options here? The form tabs are necessary otherwise you'd get a gigantic form that you'd have to check every field for. We have to have a place for errors that involve more than one field, so it should be fine to use that, so that should be fixed if it really is a problem rather than just addressing this single field.

Reply:
> I think this is too fragile. It relies on the fact that the option is on the first tab and that the tab is displayed by default. For these settings all validation errors go to the top otherwise they're hidden. This is not just a problem for parent selection.

Crispy forms have tab focus at the time of errors... but having typed that I realize there isn't any indication of multiple errors across multiple so it's still not a sufficient replacement. I agree.

We want to display the problems that might exist on a per-phase basis. When a challenge organizer starts with the challenge there are a heap of warnings, and errors, ranging from submissions not being open to phase family links.

I'll create an issue (maybe later a pitch) but I'll remove the try except here for now.